### PR TITLE
Add issueManagement to the pom file

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: plugin/pom.xml [hunk 1]
```diff
@@ -93,4 +93,9 @@
     <url>https://github.com/jenkinsci/configuration-as-code-plugin</url>
     <tag>${scmTag}</tag>
   </scm>
+  
+  <issueManagement>
+    <system>GitHub</system>
```
Comment (oleg-nenashev) [inline plugin/pom.xml]:
```suggestion
    <system>GitHub Issues</system>
```

**Comment by casz** [comment]
Any hints what work this is connected to?

**Comment by timja** [comment]
See the mailing list, working on adding issue tracker into the plugin site and encouraging more GitHub issue usage rather than jira

**Comment by timja** [comment]
Closing in favour of https://github.com/jenkins-infra/update-center2/pull/331 which doesn't require pom changes

**Pull request closed without merging**